draft-js-mention-plugin/src/mentionSuggestionsStrategy.js

```diff
@@ -1,9 +1,8 @@
 /* @flow */
 
 import findWithRegex from 'find-with-regex';
+import _ from 'lodash';
```

Comment:
Just noticed this as well. This problematic as it would include the whole lodash library. By using `import escapeRegExp  from 'lodash/escapeRegExp'` we can reduce the imported code and skip all other lodash code not necessary for this plugin.